drivers/entropy/entropy_nrf5.c

```diff
@@ -278,12 +278,15 @@ static int entropy_nrf5_get_entropy_isr(struct device *dev, u8_t *buf, u16_t len
 
 	while (len) {
 		NRF_RNG->EVENTS_VALRDY = 0;
-		NRF_RNG->TASKS_START = 1;
+		nrf_rng_task_trigger(NRF_RNG_TASK_START);
 		while (NRF_RNG->EVENTS_VALRDY == 0) {
+#if defined(CONFIG_SOC_SERIES_NWTSIM_NRFXX)
```

Comment:
This macro doesn't exist in upstream Zephyr, can we do this with static inline instead?

Reply:
Shouldn't we actually have the WFE for all platforms instead of a spinning busy wait?

Reply:
This is the `_isr` variant which is callable from ISR. One doesn't wait for interrupts in an ISR, so busy-loop instead